### DISCUSSION TITLE
How to exclude headers in %TRACK_TITLE%?

### DISCUSSION THREAD

**Discussion by vincentvandercruyssen** [ghDaYuYu/foo_discogs#88] [Help wanted] [answered]
Hi
Any idea how I make sure the headers are excluded from the track titles when tagging? my tag map for TITLE is currently $trim(%TRACK_TITLE%)

for example when tagging following album
[Ryoji Ikeda – 1000 Fragments (1995, CD) - Discogs](https://www.discogs.com/release/103915-Ryoji-Ikeda-1000-Fragments)
title-tags get formatted like this:
1. Channel X (1985-95) (Testone)
2. Channel X (1985-95) (Trans-Missions)
3. Channel X (1985-95) (This Is A Recording)
4. Channel X (1985-95) (Radiorange)
5. ...

I'd want them to be:
1. Testone
2. Trans-Missions
3. This is a Recording
4. Radiorange
5. ...

what attribute should I be using instead?

**Comment by ghuDaYuYu** [comment, marked as answer]
Hi,
you can try %TRACK_SUBTRACK_TITLE%.
It is also available from the Tag mapping panel as DISCOGS_SUB_TRACK_TITLE ('Add tag...' button -> track -> DISCOGS_SUB_TRACK_TITLE).

**Comment by vincentvandercruyssen** [comment]
Thanks!
